### PR TITLE
MAID-2337 feat/Expose a set of constants which can be used by client apps

### PR DESCRIPTION
**Pull request by bochaco**

### CODE REVIEW

File: test/mutable.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const crypto = require('crypto');
 const should = require('should');
 const h = require('./helpers');
-const consts = require('../src/consts');
+const consts = require('../src/consts').pubConsts;
```
Comment (krishnaIndia) [inline test/mutable.js]: Can be refactored as `const {pubConsts: consts} = require('../src/consts')`

Comment (krishnaIndia) [reply test/mutable.js]: We can bring the import refactoring as a separate task

Comment (bochaco) [reply test/mutable.js]: I'll include those changes in this PR

**Comment by bochaco** [comment]
@krishnaIndia , the changes in the example are already part of this PR: https://github.com/maidsafe/safe_app_nodejs/pull/159/files#diff-1553b81646f62e5efcd69a7f8aeab08aL33

File: src/api/emulations/nfs.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const lib = require('../../native/lib');
 const t = require('../../native/types');
 const nativeH = require('../../native/helpers');
-const consts = require('../../consts');
+const consts = require('../../consts').pubConsts;
```
Comment (joshuef) [inline src/api/emulations/nfs.js]: Let's import `consts` as `CONSTANTS` for consistency

File: src/api/mutable.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ const lib = require('../native/lib');
 const t = require('../native/types');
 const emulations = require('./emulations');
 const { SignKey } = require('./crypto');
-const CONST = require('../consts');
+const CONST = require('../consts').pubConsts;
```
Comment (joshuef) [inline src/api/mutable.js]: similarly here, `CONSTANTS` I think is clearest.

**Pull request merged**